client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -0,0 +1,351 @@
+    setRequestTypesError(getInputRequestTypeNameError(requestTypeName))
+    return true;
+  }
+
+  const onDescriptionChange = (newDescription: string) => {
+    setDescription(newDescription)
+    setDescriptionError(getDescriptionError(description))
+  }
+
+  const onImageChange = (newImage: string) => {
+    setImageError(getImageError(newImage))
+    if (images.find((imageUrl) => imageUrl === newImage)) {
+      setImage(newImage)
+    }
+  }
+
+  const onSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    const tempNameError = getNameError(name);
+    const tempDescriptionError = getDescriptionError(description);
+    const tempImageError = getImageError(image);
+    const tempRequestTypesError = getRequestTypeNamesError(requestTypeNames);
+
+    if (tempNameError === "" && tempDescriptionError === "" && tempImageError === "" && tempRequestTypesError === "") {
+      // do mutation
+
+      props.handleClose()
+    }
+    else {
+      setNameError(tempNameError);
+      setDescriptionError(tempDescriptionError);
+      setImageError(tempImageError);
+
+      setRequestTypesError(tempRequestTypesError);
+    }
+  }
+
+  const handleClose = () => {
+    // id: ID
+    // name: String
+    // description: String
+    // image: String
+    // requestTypes: [ID]
+    if (changeMade) {
+      setShowAlertDialog(true);
+    }
+    else {
+      props.handleClose();
+    }
+  }
+
+
+  return <div className="request-group-form">
+    <FormModal
+      class="request-group-form-modal"
+      show={true}
+      handleClose={handleClose}
+      title={props.operation === "create" ? "Create Request Group" : "Edit Request Group"}
+      size="large">
+      {showAlertDialog &&
+        <AlertDialog
+          dialogText="You have unsaved changes to this group."
+          onExit={props.handleClose}
+          onStay={() => { setShowAlertDialog(false) }} />
+      }
+      <form onSubmit={onSubmit}>
+        <div className="request-group-form-modal-content">
+          <div className="request-group-form-modal-panel" id="left">
+            <div className="text-field-form-item">
+              <FormItem
+                formItemName="Group Name"
+                errorString={nameError}
+                isDisabled={false}
+                tooltipText="Groups describe the overall category of an item, such as stroller, crib, or bed."
+                inputComponent={<TextField
+                  name="name"
+                  placeholder="Enter a group name"
+                  type="text"
+                  input={name}
+                  isDisabled={false}
+                  isErroneous={nameError !== ""}
+                  onChange={onNameChange}
+                />
+                }
+              />
+            </div>
+            <div className="tag-input-form-item">
+              <FormItem
+                formItemName="Item Types"
+                instructions="If no types are applicable,  create a universal type such as “One Size”"
```

Comment:
There is an extra space in between "," and "create"

Reply:
whooooops, thanks!